metrotranscom/doorway
# feat: senior veteran community type

This PR addresses #660 


## Description
Creates the new senior veteran community type translations allowing us to run the community type endpoint

## How Can This Be Tested/Reviewed?
create new community type `seniorVeterans`
verify you can add it to listings on the partner site, and that they show up properly on the public site

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
